### PR TITLE
[zh] Translation of the kube-proxy configuration struct

### PR DESCRIPTION
**Pull request by tengqm**
/language zh

### CODE REVIEW

File: content/zh/docs/reference/config-api/kube-proxy-config.v1alpha1.md [hunk 1]
```diff
@@ -1,601 +1,820 @@
...
-<tr><td><code>contentType</code> <B>[Required]</B><br/>
+<tr><td><code>contentType</code> <B><!--[Required]-->[必需]</B><br/>
 <code>string</code>
 </td>
 <td>
-   contentType is the content type used when sending data to the server from this client.</td>
+   <!--
+   contentType is the content type used when sending data to the server from this client.
+   -->
+   contentType 是从此客户端向服务器发送数据时使用的内容类型（Content Type）。）
```
Comment (howieyuen) [inline content/zh/docs/reference/config-api/kube-proxy-config.v1alpha1.md:707]: extra right bracket '）'

File: content/zh/docs/reference/config-api/kube-proxy-config.v1alpha1.md [hunk 2]
```diff
@@ -1,601 +1,820 @@
...
+## `VModuleConfiguration`     {#VModuleConfiguration}
+   
+<!-- 
+(Alias of `[]k8s.io/component-base/config/v1alpha1.VModuleItem`)
+
+**Appears in:**
+-->
+（`[]k8s.io/component-base/config/v1alpha1.VModuleItem` 的别名）
+
+VModuleConfiguration 是一组文件名或文件名模式，及其对应的日志详尽程度阈值配置。
```
Comment (howieyuen) [inline content/zh/docs/reference/config-api/kube-proxy-config.v1alpha1.md:819]: missing original en text

**Review by howieyuen — CHANGES_REQUESTED** [review]

**Comment by howieyuen** [comment]
I have to say, every time I review @tengqm PR, just like read the doc, not to correct but to study.
Good Job 👍

**Review by tanjunchen — COMMENTED** [review]
/approve

**Pull request merged**